cmdstanpy/stanfit.py

```diff
@@ -1134,9 +1132,10 @@ def stan_variable(self, name: str) -> np.ndarray:
         shape = ()
         if len(col_idxs) > 0:
             shape = self._metadata.stan_vars_dims[name]
-        return np.ndarray(
-            np.array(xs).reshape(shape)
-        )
+        print('xs: {}'.format(xs))
+        print('len xs: {}'.format(len(xs)))
+        print('shape: {}'.format(shape))
```

Comment:
This seems like debugging code that was left in

Reply:
fixed